draft-birkholz-scitt-software-use-cases.md

```diff
@@ -167,6 +167,35 @@ There is no standardized way to:
 * to check for the trustworthiness of a provider
 * to check the integrity of modifications or transformations done by a provider
 
+## Post-Boot Firmware Provenance
+
+In contrast to operating systems or user space software components of a large and complex systems, firmware components are often already executed during boot-cycles before there is an opportunity to authenticate them.
+
+Authentication takes place, for example, by validating a signed artefact against a Reference Integrity Manifest (RIM).
```

Comment:
I know we have folks across both sides of the pond, but shall we agree on which way we'll spell artefact | artifact?
For reference: [Artefact vs. Artifact](https://www.grammar.com/artefact_vs._artifact)

Reply:
That's actually a typo. It should be artifact.